src/main/java/com/google/sps/servlets/DeleteAll.java

```diff
@@ -0,0 +1,32 @@
+package com.google.sps.servlets;
+
+import com.google.sps.data.DatastoreService;
+//import com.google.sps.data.Day;
```

Comment:
Remove the comments?

Reply:
Yup, thanks. Removed innecesary commented code! :)